playlists/excel.yaml

```diff
@@ -746,10 +746,19 @@
   group: Preview APIs
   api_set:
     ExcelAPI: 1.8
+- id: excel-pivottable-calculations
+  name: PivotTable Calculations
+  fileName: pivottable-calculations.yaml
+  description: Change the base calculations the PivotTable is performing.
```

Comment:
Does "base calculations" have a special meaning when it comes to PivotTables, or is it sufficient to just say "calculations"?   

Also, suggest changing "...the PivotTable is performing." to "...that the PivotTable performs."